packages/blocks/src/store/reducer.js

```diff
@@ -20,7 +20,26 @@ import { combineReducers } from '@wordpress/data';
 import { __ } from '@wordpress/i18n';
 
 /**
- * Module Constants
+ * @typedef WPBlockTypeCategory
```

Comment:
I'm on the fence about whether to call this `WPBlockTypeCategory` or `WPBlockCategory`. I think technically `WPBlockTypeCategory` is the more accurate name†, but (a) it's verbose and (b) we're already pretty consistent elsewhere in this module with calling things "block something" where we actually mean "block type something"
 ("block collections", "block names", "block styles", etc). 

See also: https://github.com/WordPress/gutenberg/pull/19279#discussion_r421705366

† Accuracy as in: Consider that we don't call `labels` of `register_post_type` as "post labels". It would be strange to think about it this way, as it would be very easy to confuse this as "the labels of a post", which is meaningless. That's why the function is called [`get_post_type_labels`](https://developer.wordpress.org/reference/functions/get_post_type_labels/), and not `get_post_labels`.